website/content/en/docs/concepts/settings.md

```diff
@@ -53,7 +53,7 @@ data:
   aws.clusterCABundle: "LS0tLS1..."
   # [REQUIRED] The kubernetes cluster name for resource discovery
   aws.clusterName: karpenter-cluster
-  # [REQUIRED] The external kubernetes cluster endpoint for new nodes to connect with
```

Comment:
Can we say something like: "If not specified, will discover the cluster endpoint using DescribeCluster API"